src/scripts/modules/ajax-handlers/sbFactory.js

```diff
@@ -142,6 +142,13 @@ class SBFactoryAjaxHandler extends AjaxSuccessHandler {
             'Rift Combat Cheese': 2096,
             'Null Onyx Gorgonzola': 2100,
             'Extra Rich Cloud Cheesecake': 3274,
+            'Mythical Mulch': 3451,
```

Comment:
I would really like to get away from listing out all the cheesy outcomes but that does not have to be in this release.